component/Grids/index.html

```diff
@@ -0,0 +1,127 @@
+            <a class="nav-content"><i class="nav-logo fas fa-spa"></i></a>
+            <h2 class="nav-content nav-logo-txt">Roots</h2>
+        </div>
+        <div class="nav-child">
+            <a href="/index.html" class="nav-link nav-content">Home</a>
+            <a class="nav-link nav-content">Installation</a>
+            <a href="/component/Avatars/index.html" class="nav-link nav-content">Documentation</a>
+        </div>
+    </nav>
+    <div class="nav-body scroll">
+        <div class="nav-scroll">
+            <h4 class="nav-heading">Get Started</h4>
+            <ul class="nav-ul">
+                <li class="nav-li"><a class="nav-a">Introduction</a></li>
+                <li class="nav-li"><a class="nav-a">Colors</a></li>
+                <li class="nav-li"><a class="nav-a">Typography</a></li>
+                <li class="nav-li"><a class="nav-a">Theming</a></li>
+            </ul>
+            <h4 class="nav-heading">Components</h4>
+            <ul class="nav-ul">
+                <li class="nav-li"> <a href="/component/Avatars/index.html" class="nav-a ">Avatar</a></li>
+                <li class="nav-li"> <a href="/component/Alerts/index.html" class="nav-a ">Alerts</a></li>
+                <li class="nav-li"> <a href="/component/Badges/index.html" class="nav-a ">Badges</a></li>
+                <li class="nav-li"> <a href="/component/Buttons/index.html" class="nav-a ">Button</a></li>
+                <li class="nav-li"> <a href="/component/Cards/index.html" class="nav-a ">Card</a></li>
+                <li class="nav-li"> <a href="/component/Images/index.html" class="nav-a ">Image</a></li>
+                <li class="nav-li"> <a href="/component/Input/index.html" class="nav-a ">Input</a></li>
+                <li class="nav-li"> <a href="/component/Text-Utilities/index.html" class="nav-a ">Text-Utilities</a>
+                </li>
+                <li class="nav-li"> <a href="/component/Lists/index.html" class="nav-a ">Lists</a></li>
+                <li class="nav-li"> <a href="/component/Rating/index.html" class="nav-a ">Rating</a></li>
+                <li class="nav-li"> <a href="/component/Grids/index.html" class="nav-a ">Grids</a></li>
+                <li class="nav-li"> <a href="/component/Modal/index.html" class="nav-a ">Modal</a></li>
+                <li class="nav-li"> <a href="/component/Navigation/index.html" class="nav-a ">Navigation</a></li>
+            </ul>
+        </div>
+    </div>
+
+    <!-- GRID BODY -->
+
+
+    <div class="component-body">
+        <h2 class="component-heading">Grids</h2>
+        <h5 class="component-content"></h5>
+
+        <!-- GRID 2 COLUMN -->
+        <h4 class="component-subheading"> Grid with two column </h4>
+        <div class="container">
+            <div class="grid-two">
+                <div class="grid-box"></div>
+                <div class="grid-box"></div>
+            </div>
+        </div>
+        <div class="iframe-container">
+            <code>
```

Comment:
Instead of div using only code tag and giving it styles will work

Reply:
It's not working that's why i had to give a container. Other than this all changes are done.